### ISSUE TITLE
Broken for refracted element in meta

### ISSUE THREAD

**Issue by danielgtaylor** [refractproject/minim#94]
Looks like Minim is actually broken when using source maps, or seemingly any refracted element in metadata:

``` js
var minim = require('minim').namespace();

var el = minim.fromRefract({
  element: 'object',
  meta: {
    title: {
      element: 'string',
      content: 'Test'
    }
  }
  content: []
});

console.log(el.title);
// => { element: 'string', content: 'test' }
// Should be => 'test'

console.log(el.meta.get('title').element);
// => 'object'
// Should be => 'string'
```

The result of this bug is that Minim is basically unusable with source maps, since they break out the title/description/etc into elements and you then need to determine whether to access `el.title` or `el.title.content`. It also breaks the concept of `el.title` always returning a simple non-refracted type.

Interestingly, `el.toRefract()` seems to output the correct data.

cc @smizell

**Comment by pksunkara** [comment]
@danielgtaylor Do you have any idea why this is happening only with `meta` and not with `attributes`?

**Comment by smizell** [comment]
@pksunkara is this is something you've resolved with the recent changes?

**Comment by pksunkara** [comment]
@smizell @danielgtaylor Fixed in minim 0.14.0
